spec/spec-bootstrap.coffee

```diff
@@ -1,27 +1,15 @@
-# Start the crash reporter before anything else.
-require('crash-reporter').start(productName: 'Atom', companyName: 'GitHub')
+require '../src/window'
```

Comment:
I think these requires and the `Atom.loadOrCreate` call should still be in a try block so it exits on caught errors or shows the window, unless something else has changed where that is no longer an issue.